deeprank2/neuralnets/gnn/alignmentnet.py

```diff
@@ -172,7 +203,26 @@ def run_through_network(self, edges, edge_attr, node_attr, with_output_attention
         return self.modlist[-1].output(node_attr, True)  # (boolean-positional-value-in-call)
 
 
-class AlignmentGNN(SuperGNN):  # noqa: D101
+class AlignmentGNN(SuperGNN):
+    """Graph Neural Network.
```

Comment:
This sounds quite generic to me. Isn't the AlignmentGNN a specific type of GNN?

Reply:
Difficult to say since there are no references. In general, this docs have the bare minimum. Again I think they could be improve quite substantially, but it's out of scope for us at this stage.